make/InitSupport.gmk

```diff
@@ -313,6 +313,11 @@ else # $(HAS_SPEC)=true
       SOURCE_DATE := $$(shell $$(DATE) +"%s")
     endif
     export SOURCE_DATE_EPOCH := $$(SOURCE_DATE)
+    ifeq ($$(IS_GNU_DATE), yes)
+      export SOURCE_DATE_ISO_8601 := $$(shell $$(DATE) --utc --date="@$$(SOURCE_DATE_EPOCH)" +"%Y-%m-%dT%H:%M:%SZ" 2> /dev/null)
+    else
+      export SOURCE_DATE_ISO_8601 := $$(shell $$(DATE) -u -j -f "%s" "$$(SOURCE_DATE_EPOCH)" +"%Y-%m-%dT%H:%M:%SZ" 2> /dev/null)
+    endif
```

Comment:
This is starting to look pretty good, but I would ask you to try to break these long lines. We try to keep the makefiles reasonably narrow for easier future 3-way merges and diff viewing.

Reply:
ah yes, fixed